apps/events/models.py

```diff
@@ -541,6 +541,11 @@ class AttendanceEvent(models.Model):
 
     payments = GenericRelation("payment.Payment")
 
+    @property
+    def id(self):
```

Comment:
```suggestion
    def id(self) -> int:
```

Reply:
Yeah, but this will be inferred by most editors?

Reply:
Probably? But no reason not to add it